unittests/tools/test_rusty_hog_parser.py

```diff
@@ -0,0 +1,41 @@
+from ..dojo_test_case import DojoTestCase
+from dojo.tools.rusty_hog.parser import RustyhogParser
+from dojo.models import Test
+
+
+class TestRustyhogParser(DojoTestCase):
```

Comment:
The test cases shouldn't only check the number of findings but some or all of the expected contents as well.